### PR TITLE
Nc news6

### PR DESCRIPTION
**Pull request by charlieBdev**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -116,4 +117,41 @@ describe('GET /api/articles', () => {
...
+            body.forEach((comment) => {
+                expect(comment).toHaveProperty('comment_id'), expect.any(Number)
+                expect(comment).toHaveProperty('votes'), expect.any(Number)
+                expect(comment).toHaveProperty('created_at'), expect.any(String)
+                expect(comment).toHaveProperty('author'), expect.any(String)
+                expect(comment).toHaveProperty('body'), expect.any(String)
+                expect(comment.article_id).toBe(1)
+            })
+            expect(body).toBeSortedBy('created_at', { descending: true })
+        })
```
Comment (pippaamy) [inline __tests__/app.test.js:133]: can we add a 200 test for if the article id exists but there are no comments 🧐

**Review by pippaamy — COMMENTED** [review]
Great work, one more test to add! can you pop that in then do another pr :)

**Review by pippaamy — COMMENTED** [review]
good changes, well done!!

File: __tests__/app.test.js (hunk 1 above)
Comment (charlieBdev) [reply __tests__/app.test.js:133]: all done, thanks

**Pull request merged**